index.html

```diff
@@ -56,12 +56,12 @@ <h2>
 
   <div>
     <h2>
-      <a href="https://standard.publiccode.net/">
+      <a href="/standard-for-public-code/">
         We maintain the Standard for Public Code
       </a>
     </h2>
     <p>The Standard for Public Code is a model for public organizations building open source solutions to enable successful reuse by other public organizations.</p>
-    <p><a href="https://standard.publiccode.net/">It is here for you to use</a></p>
+    <p><a href="/standard-for-public-code/">It is available for you to use</a></p>
```

Comment:
```suggestion
    <p><a href="/standard-for-public-code/">It is ready for you to use</a></p>
```

Reply:
What about the original 'here'? Does that seem to inaccurately reference a physical location? ("Available" doesn't sound very welcoming or encouraging).

Reply:
I think "here" seems fine enough.

Reply:
I think this whole line can easily be omitted since the h2 already contains a link. I don't think this adds anything. The link in the heading should ideally be reduced to only be the words 'Standard for Public Code' to more clearly illustrate what it links to.

Reply:
But it does not link to the Standard. It links to a page about the Standard.